pages.pt_PT/linux/archinstall.md

```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Instalador com instruções para Arch Linux 
+> Mais informações em: <https://archinstall.readthedocs.io>.
+
+- Iniciar o instalador guiado:
```

Comment:
Please use imperative mood:
```suggestion
- Inicia o instalador guiado:
```

Reply:
I would keep it in the infinitive form as the remaining descriptions.